packages/api/src/brevo-contact/brevo-contact-import.service.ts

```diff
@@ -0,0 +1,169 @@
+import { parseString } from "@fast-csv/parse";
+import { Inject, Injectable } from "@nestjs/common";
+import { IsEmail, IsNotEmpty, validateOrReject } from "class-validator";
+
+import { isErrorFromBrevo } from "../brevo-api/brevo-api.utils";
+import { BrevoApiContactsService, CreateDoubleOptInContactData } from "../brevo-api/brevo-api-contact.service";
+import { BrevoContactsService } from "../brevo-contact/brevo-contacts.service";
+import { BrevoModuleConfig } from "../config/brevo-module.config";
+import { BREVO_MODULE_CONFIG } from "../config/brevo-module.constants";
+import { TargetGroupInterface } from "../target-group/entity/target-group-entity.factory";
+import { TargetGroupsService } from "../target-group/target-groups.service";
+import { EmailCampaignScopeInterface } from "../types";
+
+class ValidateableRow {
+    @IsEmail()
+    @IsNotEmpty()
+    email: string;
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    [key: string]: any;
+}
+
+@Injectable()
+export class BrevoContactImportService {
+    constructor(
+        @Inject(BREVO_MODULE_CONFIG) private readonly config: BrevoModuleConfig,
+        private readonly brevoApiContactsService: BrevoApiContactsService,
+        private readonly brevoContactsService: BrevoContactsService,
+        private readonly targetGroupsService: TargetGroupsService,
+    ) {}
+
+    async importContactFromCsv<T extends ValidateableRow>(
```

Comment:
```suggestion
    async importContactsFromCsv<T extends ValidateableRow>(
```

for me, this makes more sense in plural because several contacts are imported from one file

Reply:
7b1fa97de4d56fa04f48192c9b3c3712a68fb6e6